.github/workflows/run-integration-tests.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.11"]
-        agent: ["SWEAgent", "PlannerAgent", "MonologueAgent", "CodeActAgent"]
+        agent: ["ManagerAgent", "SWEAgent", "PlannerAgent", "MonologueAgent", "CodeActAgent"]
```

Comment:
Let's remove SWEAgent and MonologueAgent from the eval set. I'd say both of those are deprecated.

Planner is good for testing the planning functionality, but maybe we remove that one too...

Reply:
Will do in https://github.com/OpenDevin/OpenDevin/pull/1888

MonologueAgent might still be useful since it’s currently the only agent that supports memory condensation.

Reply:
I also feel like Planner should be deprecated. It's not under active development and it doesn't seem to work well.